packages/wp-now/src/node-pool.ts

```diff
@@ -0,0 +1,44 @@
+import { PHPResponse } from '@php-wasm/universal';
+import { NodePHP } from '@php-wasm/node';
+import { Pool } from './pool';
+
+export type nodePoolOptions = {
+	spawn?: () => Promise<NodePHP>;
+	reap?: (instance: NodePHP) => void;
+	fatal?: (instance: NodePHP, error: any) => any;
+	maxRequests?: number;
+	maxJobs?: number;
+};
+
+const defaultSpawn = async () => await NodePHP.load('8.2');
```

Comment:
Let's avoid hardcoding specific PHP versions and use the latest supported version. `LatestSupportedPHPVersion` should be available as an export in one of the `@php-wasm` packages.

Reply:
@adamziel I was thinking about this, but if wp-now is started with a different PHP version, I think we should continue to use that. Do you know how I can grab the value passed on the command line here?

Reply:
@adamziel Ah wait, this is just the default spawner, the calling code handles that.

Reply:
> but if wp-now is started with a different PHP version, I think we should continue to use that
> Do you know how I can grab the value passed on the command line here?

I'm not sure I follow, it seems like `wp-now` provides a custom `spawn` callback that uses the PHP version passed as the CLI arg?